.changeset/twelve-otters-swim.md

```diff
@@ -0,0 +1,5 @@
+---
+"@thirdweb-dev/react-native-adapter": minor
+---
+
+initial release
```

Comment:
this means we start on `0.1.0` is this wanted?

Reply:
dont feel strongly - either 0.1.0 or 0.0.1

Reply:
yeah I think 0.1.0 makes sense or 1.0.0